### DISCUSSION TITLE
Login to existing user using OIDC

### DISCUSSION THREAD

**Discussion by kaibagley** [IgnisDa/ryot#900] [Q&A] [answered]
Hey guys,

Might have missed a guide somewhere as someone in an issue mentioned they migrated their users to OIDC, but can anyone help me with this?

I only have one user, but I want to start using Authentik OIDC to log in as the user. I'm just not sure how...

**Comment by IgnisDa** [comment, marked as answer]
Hey I have already replied to your other discussion with the guide. CLosing this.

EDIT: Here is the link: https://github.com/IgnisDa/ryot/pull/750#issuecomment-2046204681
